### DISCUSSION TITLE
comments on 2d vs 3d simulations

### DISCUSSION THREAD

**Discussion by nash-host** [epochpic/epoch#402] [Q&A] [answered]
Dear EPOCH community,

Can the accuracy of 2d simulations be addressed briefly against 3d? To be more specific in the context of TNSA, the consequence of reducing the dimensions to two on:
1. Laser pulse and corresponding effects missed/added in 2d in comparison to 3d.
2. The energy conversion from laser to electrons.
3. The proton distribution.
4. The electric and magnetic field strengths.

Assuming same resolution and number of particles per cell in both 2d and 3d, what are the effects on the physics of the interaction solely due to the dimensionality?
any study material addressing the deviations of 2d simulations from 3d is greatly appreciated.

Thanks in advance :)

**Comment by DanRRRR** [comment, marked as answer]
The answer is easy. With 2D you might lose some relevance to real 3D world situation but might gain by reducing the noise in simulations and perform simulations faster. The noise as we all know decreases as square root of number of particles per cell (same way like in photography matrix noise depends on how many electrons can hold your pixel. Quantum noise has the same nature too)

As to the TNSA you might overestimate the energy you will be getting in 2D because one of your dimensions implied to be infinite in size while in 3D reality the acceleration length is of the order of focal spot or so. I guess the major differences will be that your peak (cut-off) energies in 2D could be larger then in 3D and angular dependence in 2D narrower.  You can do the experiment yourself using existing EPOCH examples. Let us know what you get

There exist dramatic examples in other fields when you go from 1d to 2d and then 3d and see how your simulation results change not just quantitatively but qualitatively. In one dimension your simulation  get you something which looks like a new physical phenomena, but with 3d it diminishes to almost a dud ;)

**Comment by nash-host** [comment]
Many thanks Dan for your response :)

Sure, I will try with 1d and 2d simulations and let you know the comparison.
I was under the impression that 2d simulations somewhat serves the pusrpose for qualitative analysis. I realized it is not so straight forward.

If possible, could you please suggest me a paper or something where specific parameters are compared with regard to different dimensions?
Thank you very much :) :)

**Reply by DanRRRR** [reply]
Sorry, i could not find that example where i suspected it was. Hopefully will find more time to look deeper closer to holidays.
